lib/nfg_ui/bootstrap/components/base.rb

```diff
@@ -65,8 +65,13 @@ def html_options
                                               # Example: <div class>Text</div>
         end
 
+        # Use view_context urL_for method to ensure that
+        # when objects are passed into href, e.g.
+        # `href: admin_path(admin)` will convert it to the
+        # correct path
         def href
-          options[:href]
+          return if options[:href].nil?
+          view_context.url_for(options[:href])
```

Comment:
consider extracting into an `href` variable so you save a lookup in the options hash

Reply:
@subelsky can you clarify / explain? Not totally sure what you mean here. Sorry if I'm being dense :)

Reply:
Oh do you mean `href = options[:href]` so we're not diving into options twice / three times.

Reply:
yup that's it

Reply:
it's possible ruby can cache the lookup. I also think it's worth doing because it makes the code a little more brief & so the important data item stands out a bit

Reply:
Awesome. thanks.